pyspectrafuse/cluster_parquet_combine/combine_cluster_and_parquet.py

```
@@ -20,55 +20,57 @@ def __init__(self):
 
-    def inject_cluster_info(self, path_parquet, clu_map_dict, path_sdrf, spectra_num=1000000, batch_size=200000):
+    def inject_cluster_info(self, path_parquet: Union[str, List[str]], 
+                           clu_map_dict: Dict[str, str], path_sdrf: str,
+                           spectra_num: int = 1000000, batch_size: int = 200000) -> pd.DataFrame:
         # read parquet file and the cluster result tsv file of Maracluster program.
-        print("inject cluster_info: path_parquet:", path_parquet)
-        if type(path_parquet) == list:
+        logger.info(f"Injecting cluster info: path_parquet: {path_parquet}")
+        if isinstance(path_parquet, list):
+            if not path_parquet:
+                raise ValueError("path_parquet list is empty")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)